service/lib/dinstaller/language.rb

```diff
@@ -37,15 +37,25 @@ def initialize(logger)
       @languages = []
     end
 
+    # Probes the available system languages
+    #
+    # @param _progress [Progress] Progress reporting object
     def probe(_progress)
```

Comment:
Question: why those methods receive a param that are not used?

Reply:
For consistency with the rest of `#probe` and `#install` methods.